modules/ROOT/pages/release-notes.adoc

```diff
@@ -5,3 +5,33 @@
 ====
 The 2023.1 is in development.
 ====
+
+== Improvements
+
+=== Development suite changes
+
+==== Bonita project
+
+Bonita projects layout has been updated to take advantage of Maven multi-module project perks.
+
+== Feature deprecations and removals
+
+=== Custom Rest API Extension and BDM dependencies
+
+When updating a Bonita project in the 2023.1 version, BDM dependencies are updated to use the new the `*-bdm-model` artifact introduced with the new Bonita project layout changes. This automatic update will replace previous `bdm-client` and `bdm-dao` dependencies with the project `bdm-model` dependency. 
+In the process the dependency containing the client DAO implementation is removed from the dependencies list. 
+
+It can affects your code if you were directly referencing the `*DAOImpl` classes. If you find yourself in this situation we invite you to update your to use the DAO interfaces presents in the `bdm-model` dependency. 
```

Comment:
```suggestion
It can affects your code if you were directly referencing the `*DAOImpl` classes. If you find yourself in this situation we invite you to update this code to use the DAO interfaces presents in the `bdm-model` dependency. 
```